acceptance/tests/pxp_agent_version.rb

```diff
@@ -1,12 +1,17 @@
 test_name 'C93805 - pxp-agent - Versioning test'
 
-agent1 = agents[0]
+def pxp_version_cmd(platform)
+  if platform.start_with?('windows')
+    # simple hack to ensure OpenSSL is found w/out modifying PATH
+    "cd /cygdrive/c/Program\\ Files/Puppet\\ Labs/Puppet/sys/ruby/bin && ../../../pxp-agent/bin/pxp-agent --version"
```

Comment:
The case of 32bit Ruby where Puppet is in C:\Program Files (x86) needs to be supported here